Notebooks/load_multiple_wells.ipynb

```diff
@@ -54,58 +54,226 @@
+    "         'POTA': ['KCPS', 'HPOT', 'POTA', 'K'],\n",
+    "         'RESD': ['RESD','RT_H','RLLD','HATR','HAT9','ASTR','AST9','AS90','AIT9','AHTR','AHT9','ID40','VRSD','VP65','RT_HRLT','RT','RLA3','M4RX','M1RX',\n",
+    "                  'M0RX','ILD2','IDVR','IDPH','HRLD','FE2','DVR2','DIPH','ATRT','AT90','AST90','AIT90','AIT120','AHTRT','A22H','A22H_','A22H_UNC','A28H',\n",
+    "                  'A28H_','A28H_UNC','A34H','A34H_','A34H_UNC','AF60','AF90','AFRT', 'AHF60','AHF90','AHO60','AHO90','AHT60','AHT90','AILD','AIT','AIT60',\n",
+    "                  'AO12','AO60','AO90','AORT','ASF60','ASF90','ASO60','ASO90','AST60','AT60','DVR1','DVR4','GRD','HILD','HLLD','HRL4','HRL5','IDER','IL',\n",
+    "                   'ILD','ILD4','LL','LL3','LL7','LLD','LN','M0R6','M0R9','M1R6','M1R9','M2R6','M2R9','M2RX','M4R6','M4R9','P10H','P10H_','P10H_UNC','P16H',\n",
+    "                   'P16H_','P16H_UNC','P22H','P22H_','P22H_UNC','P28H','P28H_','P28H_UNC','P34H','P34H_','P34H_UNC','RFOC','RILD','RIPD','RLA1','RLA2','RLA4',\n",
+    "                   'RLA5','RLL3','RLL4','RLL5','VILD','AIT6','DDLL','EIRD','ID10','ID20','ID25','ILD1','LLG','REID','RLL','AHF6','AHF9','AHO6','AHO9','AHOR',\n",
+    "                   'AHT6','AS60','ASF6','ASF9','ASFR','ASRT','AST6','HAT6'],\n",
+    "         'RESM': ['RESM','RLLS', 'HAT3','AST3','AS30','AHT3','VP25','RIPM','RILM','MVR2','M4R3','M2R3','M1R3','M0R3','IMVR','IMPH','IMER','IM','ILM2','ILM','HRL3','HLLS',\n",
+    "                  'HILM','AT30','AST30','AIT30','AHT30','AF30','AHF30','AHO30', 'AO30', 'ASF30','ASO30','ILM4','LLS','MVR1','MVR4','VILM','AE60','ILD25','ILM1','ILM25','IM10',\n",
+    "                  'IM20','IM25','IM40','PIRM','REIM','DSLL','AIT3','AHF3', 'AHO3','ASF3','RMLL'],\n",
+    "         'RESS': ['RESS', 'RXOH','MSFF','HSFL','HAT1','AST1','AS10', 'AHT1','RXOZ','RXOI','RXO8','RXO1D','RXO_HRLT','RXO','RLL1','RESS','HRL1','ATRX','AHTRX','AF10','AF20','AFRX',\n",
+    "                  'AHF10','AHF20','AHO10','AHO20','AHSFI','AHT10','AHT20','AIT10','AIT20','AO10','AO20','AORX','ASF10','ASF20','ASFI','ASFL','ASN','ASO10','ASO20','AST10','AST20',\n",
+    "                  'AT10','AT12','AT20','FE1','FEFE','FR','FRA','HRL2','HRLS','HSFLU','ILS','LL8','LLA','M0R1','M0R2','M1R1','M1R2','M2R1','M2R2','M4R1','M4R2','MFR','MLL','MSFL',\n",
+    "                  'PSHG','RES','RILS','RLL2','RS','RSFE','RSFL','RSG','SFBC','SFL','SFL4','SFLA','SFLR','SFLU','SN','AE20','AE30','AHF1','AHF2','AHO1','AHO2','AHT2', 'AS20','ASF1','ASF2','ASRX','AST2','HAT2'],\n",
+    "         'SP': ['SP_S','AHSF','SPSB','SPH','SPDH','SP','HSP','PCH1','SPBR','SPC','SPCG','SPD','SPDF','SPP', 'SSPK','SPS','AHSC','SPA_'],\n",
+    "         'THOR': ['TCPS', 'THOR', 'HTHO', 'TH'],\n",
+    "         'URAN': ['UZ', 'UCPS', 'URAN', 'U', 'HURA']\n",
+    "        }\n",
+    "keys= ['CAL','DENS','DCOR','DT','GR','PE','PHID','PHIN','PHIS','POTA','RESD','RESM','RESS','SP','THOR','URAN']"
```

Comment:
These are the keys from alias if I am not mistaken, to help prevent bugs it would be better to either us `alias.keys()` in the future or assign the variable keys to `keys=alias.keys()` in case any other values get added or removed